### PR TITLE
feat: creates goenvtemplator2 binary

### PR DESCRIPTION
**Pull request by dohnto**
@lksv @mody @rembish

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -3,11 +3,11 @@ Maintainer: https://github.com/seznam/goenvtemplator
 Section: admin
 Priority: optional
 Build-Depends: debhelper (>= 9), golang (>= 2:1.5), git
-Replaces: goenvtemplator
-Breaks: goenvtemplator
 
 Package: goenvtemplator2
 Architecture: amd64
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Built-Using: ${misc:Built-Using}
```
Comment (mody) [inline debian/control]: You don't need the `Built-Using:` here, see [Debian Policy](https://www.debian.org/doc/debian-policy/ch-relationships.html#s-built-using) or [Debian bug 688251](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=688251).

**Comment by rembish** [comment]
I've had many problems during automatic build in my machine. Some `go` packages didn't download automatically. `GOPATH` doesn't set at all. `lintian` isn't happy:
```
E: goenvtemplator changes: bad-distribution-in-changes-file Seznam
W: goenvtemplator source: diff-contains-git-control-dir .git
E: goenvtemplator source: source-is-missing goenvtemplator2
E: goenvtemplator source: maintainer-address-missing https://github.com/seznam/goenvtemplator
W: goenvtemplator source: no-debian-copyright
E: goenvtemplator source: no-standards-version-field
W: goenvtemplator2: hardening-no-relro usr/bin/goenvtemplator2
E: goenvtemplator2: no-copyright-file
E: goenvtemplator2: maintainer-address-missing https://github.com/seznam/goenvtemplator
W: goenvtemplator2: binary-without-manpage usr/bin/goenvtemplator2
```

File: debian/control (hunk 1 above)
Comment (dohnto) [inline debian/control]: fixed via 4ce9acb

**Comment by dohnto** [comment]
@rembish: I fixed some of these. Remaining: 
```
E: goenvtemplator changes: bad-distribution-in-changes-file Seznam
W: goenvtemplator source: diff-contains-git-control-dir .git
E: goenvtemplator source: source-is-missing goenvtemplator2
E: goenvtemplator source: source-is-missing debian/goenvtemplator/usr/bin/goenvtemplator
W: goenvtemplator source: changelog-should-mention-nmu
W: goenvtemplator source: source-nmu-has-incorrect-version-number 2.0.0~rc2
W: goenvtemplator source: no-debian-copyright
W: goenvtemplator2: hardening-no-relro usr/bin/goenvtemplator2
E: goenvtemplator2: no-copyright-file
```
I will create a separate issue for this, but I don't see a solution for many of them (e. g. bad-distribution-in-changes-file)

**Comment by dohnto** [comment]
Rebased & merged manually. Issues concerning debian packaging moved to #7. Closing

**Pull request closed without merging**